app/views/users/registrations/new.html.erb

```diff
@@ -2,28 +2,27 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">Welcome!</h1>
-    <p class="govuk-body">
-      We are glad you would like to sign up your organisation for GovWifi.
-    </p>
-    <p class="govuk-body">
-      You have the option to sign up for a three month trial. After you have
-      tested GovWifi, you can go live and scale it at your organisation for free!
+    <h1 class="govuk-heading-l">Sign up your Organisation to GovWifi</h1>
+
+    <p class="govuk-!-padding-bottom-1">
+      <%= link_to "If you are trying to connect to GovWifi as an end user, see instructions here", "", class: "govuk-link govuk-!-font-size-19" %>
```

Comment:
I would suggest just making the end bit the link. More Govuk'y. Eg:
![screen shot 2018-08-23 at 11 52 58](https://user-images.githubusercontent.com/2160769/44521479-1a9f0080-a6cb-11e8-82d5-bd80a3171f5c.png)

Reply:
Good shout:

![image](https://user-images.githubusercontent.com/429326/44521847-7a49db80-a6cc-11e8-814f-8ee279691365.png)

Commit coming shortly..